### ISSUE TITLE
No support for MySQL enum and float field type?

### ISSUE THREAD

**Issue by virtser** [ForestAdmin/lumber#52]
Following #51, I found that MySQL enum and floar fields are simply ignored by Forest and not displayed at all.

We have the following field in our DB model which works and displays just fine using Sequelize:

```
    status: {
      type: DataTypes.ENUM,
      values: ['pending', 'listed'],
      allowNull: false,
      defaultValue: 'pending'
    },
    display_order: {
      type: DataTypes.FLOAT
    }
```

**Comment by virtser** [comment]
I found that lumber creates its own models using sequelize, but it doesn't detect well the column data types. Changing it manually solves the issue.

**Comment by SeyZ** [comment]
Hey @virtser ! Thanks for report! If you have some time, can you help Lumber supporting your data types natively by patching the following method according to your schema: https://github.com/ForestAdmin/lumber/blob/master/services/table-analyzer.js#L29

Thank you!

**Comment by arnaudbesnier** [comment]
Manual changes are a workaround but definitly not a fix.
This issue should be reopened.

**Comment by ArnaudValensi** [comment]
This is fixed in the last release (1.10.0)

**Comment by jeremymaignan** [comment]
Hi, 
I am facing the same issue. How can I fix it ? 
Thanks
